test/e2e/netman/helpers.go

```diff
@@ -30,32 +30,112 @@ func deployContract(ctx context.Context, chainID uint64, client *ethclient.Clien
 	owner := txOpts.From
 	fee := new(big.Int).SetUint64(params.GWei)
 
-	feeOracleAddr, tx, _, err := bindings.DeployFeeOracleV1(txOpts, client, owner, fee)
+	feeOracleAddr, err := deployFeeOracleV1(ctx, txOpts, client, owner, fee)
 	if err != nil {
-		return common.Address{}, nil, nil, errors.Wrap(err, "deploy fee oracle contract")
+		return common.Address{}, nil, nil, errors.Wrap(err, "deploy fee oracle")
+	}
+
+	portalAddr, err := deployPortal(ctx, txOpts, client, owner, feeOracleAddr, valSetID, validators)
+	if err != nil {
+		return common.Address{}, nil, nil, errors.Wrap(err, "deploy portal")
+	}
+
+	contract, err := bindings.NewOmniPortal(portalAddr, client)
+	if err != nil {
+		return common.Address{}, nil, nil, errors.Wrap(err, "new portal contract")
+	}
+
+	return portalAddr, contract, txOpts, nil
+}
+
+func deployFeeOracleV1(
+	ctx context.Context,
+	txOpts *bind.TransactOpts,
+	client *ethclient.Client,
+	owner common.Address,
+	fee *big.Int,
+) (common.Address, error) {
+	_, tx, _, err := bindings.DeployFeeOracleV1(txOpts, client)
+	if err != nil {
+		return common.Address{}, errors.Wrap(err, "deploy fee oracle impl")
 	}
 
 	receipt, err := bind.WaitMined(ctx, client, tx)
 	if err != nil || receipt.Status != types.ReceiptStatusSuccessful {
-		return common.Address{}, nil, nil, errors.Wrap(err, "wait mined fee oracle contract")
+		return common.Address{}, errors.Wrap(err, "wait mined fee oracle impl")
+	}
+
+	feeOracleV1ABI, err := bindings.FeeOracleV1MetaData.GetAbi()
+	if err != nil {
+		return common.Address{}, errors.Wrap(err, "get fee oracle v1 abi")
 	}
 
-	portalAddr, tx, _, err := bindings.DeployOmniPortal(txOpts, client, owner, feeOracleAddr, valSetID, validators)
+	encodedFeeOracleV1Initializer, err := feeOracleV1ABI.Pack("initialize", owner, fee)
```

Comment:
> [Variable names in Go should be short rather than long](https://go.dev/wiki/CodeReviewComments#variable-names). 

Especially, when the variable is only ever used few lines below. A better alternative would be: `feeOracleInit` or even `foi`.